translations/en/LC_MESSAGES/messages.po

```diff
@@ -19,133 +19,133 @@ msgstr ""
 "Generated-By: Babel 2.14.0\n"
 
 msgid "Access Before Assign"
-msgstr "You tried to use the variable {name} on line {access_line_number}, but you set it on line {definition_line_number}. Set a variable before using it."
+msgstr "Variable {name} used on line {access_line_number} before being set. Try setting the variable before use."
 
 msgid "Cyclic Var Definition"
-msgstr "The name `{variable}` needs to be set before you can use it on the right-hand side of the `{is}` command."
+msgstr "Variable `{variable}` used on the right-hand side of the `{is}` command before being set. Try setting the variable before use."
 
 msgid "Else Without If Error"
-msgstr "On line {line_number} you used an `{else}` but there is no `{if}` on the line before it."
+msgstr "Command `{else}` used before using an `{if}` on line {line_number}. Try writing `{if}` before writing `{else}`."
 
 msgid "Function Undefined"
-msgstr "You tried to use the function {name}, but you didn't define it."
+msgstr "Function {name} used without being defined. Try to define the function before use."
 
 msgid "Has Blanks"
-msgstr "Your code is incomplete. It contains blanks that you have to replace with code."
+msgstr "Code is incomplete. Try filling in blanks with code."
```

Comment:
Putting this on this one but it is a general remark, I don't believe this is better. It feels very abrasive and short, I miss the "human communication" that we had before, this feel very much like communication with a machine. Can we make it more conversational?